homeassistant/components/swidget/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""The swidget integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from swidget.discovery import discover_single
+from swidget.exceptions import SwidgetException
+from swidget.swidgetdevice import SwidgetDevice
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+from .coordinator import SwidgetDataUpdateCoordinator
+
+LOGGER = logging.getLogger(__name__)
+PLATFORMS: list[Platform] = [Platform.LIGHT]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up swidget from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+    try:
+        device = await discover_single(
+            host=entry.data["host"],
```

Comment:
please use `CONF_HOST` and `CONF_PASSWORD`

Reply:
Fixed